src/components/ContactsPage.js

```diff
@@ -64,6 +80,18 @@ class ContactsPage extends React.Component {
     );
   }
 
+  renderDeleteSelectedButton = () => (
+    <div className="ui segment">
+      <button
+        className="fluid ui button olive"
+        type="button"
+        onClick={this.deleteSelectedContacts}
+        disabled={!this.anyContactSelected()}
+      >Delete selected {this.numberSelectedContacts()}
```

Comment:
Rename it to numberOfSelectedContacts. Makes more sense this way. sorry for not finding this the first time :(

Reply:
no problemo